docs/how-to-guides/SPICE/creating-spice-config-files-in-spiceql.md

```diff
@@ -0,0 +1,253 @@
+                └── <Text Kernel Type>/
+                    ├── <kernel db files> 
+                    └── <kernel files>
+```
+
+Where ``<mission>`` the name of any particular mission (MRO, Odyssey, Messenger, etc.), ``<Kernel Type>`` is with a slightly different structure if the kernel type is binary (CK, SPK, etc.) or text (IK, IAK, LSK, etc.), This section focuses on the Kernel "makedb" and "db" files. 
+
+Each kernel directory (ck, spk, ik, etc.) has a file called ``kernels.[0-9]{4}.db`` where ``[0-9]{4}`` is the version of the db file. As ISIS kernel metadata is not version controlled, it uses a proprietary versioning system where the 4 numbers after the first extension delimiter is a version number.  
+
+**SPK Kernel DB file, $ISISDATA/messenger/kernels/spk/kernels.0184.db**
+
+```text
+
+    Object = SpacecraftPosition
+    RunTime = 2015-09-14T13:41:46
+
+    Group = Dependencies
+        LeapsecondKernel = $base/kernels/lsk/naif0011.tls
+    End_Group
+
+    Group = Selection
+        Time = ("2004 AUG 03 07:14:39.393205 TDB",
+                "2015 APR 30 19:27:09.351488 TDB")
+        File = $messenger/kernels/spk/msgr_20040803_20150430_od431sc_2.bsp
+        Type = Reconstructed
+    End_Group
+    End_Object
+    End
+```
+
+Some explanations: 
+
+* ``Object = SpacecraftPosition``specifies this is SPK metadata. 
+* ``Group = Dependencies`` specifies Kernels required to furnish these kernels and extract meaningful data, in this case, an LSK. Usually, this is an LSK or SCLK. Sometimes includes Kernels for other reference frames. 
+* ``Group = Selection`` specifies the criteria for selecting a particular kernel, in this case, the start and stop time range, the path of the Kernel, and the Kernel quality. 
+
+For binary kernels, these DB files are generally not handwritten, instead, the higher level information is written into a script, usually named ``makedb`` with information on required time kernels, and regexes for different kernel qualities and the kernel type. 
+
+**Makedb script, ``$ISISDATA/messenger/kernels/spk/makedb``**
+
+```
+
+  #  Generate kernel database from existing kernels
+  kerneldbgen to='kernels.????.db' type=SPK \
+              recondir='$messenger/kernels/spk' \
+              reconfilter=msgr_20040803*.bsp \
+              lsk='$base/kernels/lsk/naif????.tls'
+```
+
+Things to note: 
+
+* ``type=SPK`` describes the Kernel type, this populates the Object name in the output PVL. In this case, ``Object = SpacecraftPosition``  
+* ``lsk='$base/kernels/lsk/naif????.tls`` describes the LSK dependencies, sometimes you also see ``SCLK=<path>`` in cases where a specific SCLK is required. Populates the dependencies group, in this case ``LeapsecondKernel = $base/kernels/lsk/naif0011.tls``
+* ``reconfilter=msgr_20040803*.bsp``describes the format (using Unix globbing wildcards) for this mission's reconstructed kernels for the given kernel type (SPK). Files matching these patterns are globbed, then: 
```

Comment:
Looks like a space between `reconfilter=msgr_20040803*.bsp` and `describes` is missing.
Also there's an ending `:` but no sub-points when looking at it via the website.